webhook_notification.go

```diff
@@ -6,6 +6,8 @@ import (
 )
 
 const (
+	Check = "check"
+
```

Comment:
Any reason for the \n? It might look cleaner adding grouping this with the others.